os/fs/smartfs/smartfs_utils.c

```diff
@@ -2217,6 +2218,10 @@ int smartfs_sector_recovery(struct smartfs_mountpt_s *fs)
 
```

Comment:
At line number 156, 157, 156, 2187, the division operation (i.e.,  X / 8) should be replaced with ( X >> 3) for computation efficiency.

Reply:
Will do it in another commit